.eslintrc.js

```diff
@@ -113,6 +113,7 @@ module.exports = {
     ],
     strict: 'off',
     'guard-for-in': 'off',
+    'no-restricted-syntax': ['error', 'ForInStatement', 'LabeledStatement', 'WithStatement'],
```

Comment:
@Nnaga1 without this change, there's a lint error that generators and iterators are not allowed because they require a poly-fill (specifically https://github.com/facebook/regenerator) but that's specific to running javascript in a browser and doesn't apply to node.js so I removed the "no generators" rule with this override.